rust/hyperlane-core/src/utils.rs

```diff
@@ -50,6 +50,27 @@ pub fn announcement_domain_hash(address: H256, domain: impl Into<u32>) -> H256 {
     )
 }
 
+/// Computes the domain separator for a given domain (for eigenlayer)
+pub fn domain_separator(domain: u32, service_manager_address: H160) -> H256 {
```

Comment:
any reason not to put these functions in operator_registration?

Reply:
moved